### PR TITLE
Changes to files

### PR DESCRIPTION
**Pull request by Adritsheu**

### CODE REVIEW

File: data/inferlink/extractions/Reocin_zn_summary.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
         {
           "id": 1,
           "commodity": "https://minmod.isi.edu/resource/Q589",
-          "category": "https://minmod.isi.edu/resource/classification",
+          "category": "https://minmod.isi.edu/resource/Measured",
```
[the remark this answers is not in the document]
Comment (Adritsheu) [reply data/inferlink/extractions/Reocin_zn_summary.json:45]: This was just updated in most recent commit

File: data/inferlink/extractions/La_Negra_Ag_Cu_Pb_Zn_5-2013_OM_summary_20240124_151004.json [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                  "y_min": "",
+                  "y_max": ""
+                }
+              }
+            ]
+          },
+          "date": "2013-5"
+        }
+      ],
+      "deposit_type": "{'deposit_type':[]}"
```
Comment (namrata1012) [inline data/inferlink/extractions/La_Negra_Ag_Cu_Pb_Zn_5-2013_OM_summary_20240124_151004.json:191]: The format of deposit_type is incorrect. Please check

File: data/inferlink/extractions/La_Negra_Ag_Cu_Pb_Zn_5-2013_OM_summary_20240124_151004.json [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+                "Betty Gibbs E.M. M.SC. MMSA 01164QP",
+                "Robert E Cameron Ph.D. MMSA 01357QP"
+              ],
+              "year": 2013,
+              "month": 5,
+              "description": "The document is a technical report on the La Negra Mine project in Quer\u00e9taro, Mexico, prepared for Aurcana Corporation."
+            },
+            "page_info": [
+              {
+                "page": "page number",
```
Comment (namrata1012) [inline data/inferlink/extractions/La_Negra_Ag_Cu_Pb_Zn_5-2013_OM_summary_20240124_151004.json:46]: page number should be an integer

File: data/inferlink/extractions/La_Negra_Ag_Cu_Pb_Zn_5-2013_OM_summary_20240124_151004.json [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
```
Comment (namrata1012) [inline data/inferlink/extractions/La_Negra_Ag_Cu_Pb_Zn_5-2013_OM_summary_20240124_151004.json:134]: page number should be an integer

File: data/inferlink/extractions/La_Negra_Ag_Cu_Pb_Zn_5-2013_OM_summary_20240124_151004.json [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
```
Comment (namrata1012) [inline data/inferlink/extractions/La_Negra_Ag_Cu_Pb_Zn_5-2013_OM_summary_20240124_151004.json:178]: page number should be an integer

File: data/inferlink/extractions/La_Negra_Ag_Cu_Pb_Zn_5-2013_OM_summary_20240124_151004.json [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
```
Comment (namrata1012) [inline data/inferlink/extractions/La_Negra_Ag_Cu_Pb_Zn_5-2013_OM_summary_20240124_151004.json:90]: page number should be an integer

File: data/inferlink/extractions/Reocin_zn_summary.json [hunk 7]
```diff
@@ -76,12 +76,7 @@
           "date": "2002-03-03"
         }
       ],
-      "deposit_type": [
```
Comment (namrata1012) [inline data/inferlink/extractions/Reocin_zn_summary.json:79]: This file does not have source_id and record_id

**Pull request merged**